target/linux/mvebu/cortexa9/config-4.19

```diff
@@ -0,0 +1,11 @@
+CONFIG_ARMADA_370_CLK=y
```

Comment:
It would be helpful to mention (in the commit message) that you also move stuff between target und subtarget files here. On first look, I did not understand why you touch cortexa53/cortexa72 based on your commit message.